modules/custom/az_paragraphs/az_paragraphs_link_group/templates/paragraph--az-link-group--default.html.twig

```diff
@@ -0,0 +1,108 @@
+ *   - paragraphs--type-[type]: The current paragraphs type. For example, if the paragraph is an
+ *     "Image" it would result in "paragraphs--type--image". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - paragraphs--view-mode--[view_mode]: The View Mode of the paragraph; for example, a
+ *     preview would result in: "paragraphs--view-mode--preview", and
+ *     default: "paragraphs--view-mode--default".
+ * - view_mode: View mode; for example, "preview" or "full".
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_paragraph()
+ *
+ * @ingroup themeable
+ */
+#}
+{%
+  set classes = [
+    'paragraph',
+    'az-link-group',
+    'paragraph--type--' ~ paragraph.bundle|clean_class,
+    view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
+    not paragraph.isPublished() ? 'paragraph--unpublished',
+    'az-link-group-style-' ~ link_group.group_style|clean_class,
+    link_group.group_style|clean_class,
+  ]
+%}
+
+{% set link_attributes = create_attribute() %}
+{% set button_attributes = create_attribute() %}
+{% set dropdown_attributes = create_attribute() %}
+
+{% 
+  set button_classes = [
+    'btn',
+    'my-1',
+    link_group.button_color ? link_group.button_color,
+    link_group.button_size ? link_group.button_size,
+  ]
+%}
+
+{% 
+  set dropdown_classes = [
+    'btn',
+    'dropdown-toggle',
+    link_group.button_color ? link_group.button_color,
+    link_group.button_size ? link_group.button_size,
+  ]
+%}
+
+{% 
+  set listgroup_classes = [
+    'list-group-item',
+    'list-group-item-action'
+  ]
+%}
+
+{% if link_group.group_style ==  'buttons' %}
+  {% set link_classes = button_classes %}
+{% elseif link_group.group_style == 'dropdown' %}
+  {% set link_classes = 'dropdown-item' %}
+{% elseif link_group.group_style == 'list_group' %}
+  {% set link_classes = listgroup_classes %}
+{% endif %}
+
+{% block paragraph %}
+  <div{{ attributes.addClass(classes) }}>
+    {% block content %}
+      {% if link_group.group_style == 'dropdown' %}
+        <button {{ dropdown_attributes.addClass(dropdown_classes) }} type="button" id="az-link-group-dropdown-{{ paragraph.id() }}" data-toggle="dropdown" aria-expanded="false">{{ link_group.dropdown_title }}</button>
+        <div class="dropdown-menu" aria-labelledby="az-link-group-dropdown-{{ paragraph.id() }}">
+      {% endif %}
+      {% for key, item in paragraph.field_az_link_item %}
+        {% if key|first != '#' %}
+          <a href="{{ item.url }}" {{ link_attributes.addClass(link_classes) }}>{{ item.title }}</a>
+        {% endif %}
+      {% endfor %}
+      {% if link_group.group_style != 'buttons' %}
+        {# Close surrounding divs if dropdown or list-group #}
```

Comment:
I think this will fix the issue with the sidebar position when using the List Group style:
```suggestion
      {% if link_group.group_style == 'dropdown' %}
        {# Close surrounding div if dropdown #}
```

Reply:
Just what I was thinking! :)